<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-N2PRSIL" failures="1" tests="6" name="EdgeTest" time="72.967" errors="0" timestamp="2025-06-05T16:01:11 IST">
  <testcase classname="testCases.TC_003LoginWithInvalidEmailAndPassword" name="ClickOnSignup" time="1.328"/>
  <testcase classname="testCases.TC_001SignupForNewUser" name="ClickSignupLogin" time="6.665"/>
  <testcase classname="testCases.TC_004ContacttoUsTest" name="ContactToUs" time="8.768">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_004ContacttoUsTest.ContactToUs(TC_004ContacttoUsTest.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- ContactToUs -->
  <testcase classname="testCases.TC_002VerifyTestCases" name="VerifyTestCases" time="3.015"/>
  <testcase classname="testCases.TC_001SignupForNewUser" name="EnterAccountDetails" time="10.056"/>
  <testcase classname="testCases.TC_003LoginWithInvalidEmailAndPassword" name="VerifLoginFunctionality" time="2.443"/>
</testsuite> <!-- EdgeTest -->
